tests/unit/test_renderable_schema.py

```diff
@@ -1,12 +0,0 @@
-from consultation_analyser.consultations import public_schema
-from consultation_analyser.consultations.decorators.renderable_schema import RenderableSchema
-
-
-def test_renderable_schema():
-    renderable = RenderableSchema(public_schema.Consultation)
```

Comment:
We can also delete the RenderableSchema itself!

Reply:
Now deleted.